src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -229,54 +230,76 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
   def adminDeleteWorkspace(workspaceName: WorkspaceName): Future[PerRequestMessage] = asFCAdmin {
     dataSource.inTransaction { dataAccess =>
       withWorkspaceContext(workspaceName, dataAccess) { workspaceContext =>
-        deleteWorkspace(workspaceName, dataAccess, workspaceContext)
+        DBIO.successful(workspaceContext)
```

Comment:
actually, screw it. can you just do the above refactor? since we're trying not to incur additional techdebt an'all.